README.yaml

```diff
@@ -17,7 +17,7 @@ license: "APACHE2"
 copyrights:
   - name: "Cloud Posse, LLC"
     url: "https://cloudposse.com"
-    year: "2021"
+    year: "2022"
```

Comment:
don't change the year here, it's the starting year

Reply:
That's what I get for going above and beyond lol... now I have to rerun the tests.